bots/bridge-tester/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "bridge-tester",
+    "version": "0.0.2",
+    "description": "Bot for testing the Interlay and Kintsugi bridges.",
+    "main": "build/index.js",
+    "typings": "build/index.d.ts",
+    "repository": "https://github.com/interlay/bots/bridge-tester",
+    "author": "Interlay",
+    "license": "Apache-2.0",
+  "engines": {
+    "node": ">=11"
+  },
+  "engineStrict": true,
+
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "start": "node ./build/index.js",
+    "live": "ts-node src/index.ts",
+    "test": "run-s build test:*",
+    "test:integration": "mocha test/**/*.test.ts --timeout 10000000"
+  },
+  "dependencies": {
+    "@interlay/interbtc-api": "1.8.3",
```

Comment:
This is an interesting dependency version. I assume this is from what it used to be?

Should we expect potential issues in local tests, given that the test setup script grabs the latest `interlay/parachain-launch` version which is a few runtime upgrades ahead of what interbtc-api 1.8.3 has been built with?

Reply:
Actually this is outdated, it's the last version the `bridge-bot` was updated to. I'm using the latest in my `lending-liquidator` branch so it shouldn't cause issues.